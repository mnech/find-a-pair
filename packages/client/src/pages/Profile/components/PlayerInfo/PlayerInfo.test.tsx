import React from 'react';
import { render } from '@testing-library/react';
import { PlayerInfo } from './index';

describe('PlayerInfo', () => {
  describe('should be rendered correctly PlayerInfo component', () => {
    test('default', () => {
      const testUser = {
        first_name: 'Иван',
        second_name: 'Иванов',
        display_name: 'Иванушка',
        login: 'ivanovv',
        email: 'ivanov@yandex.ru',
        phone: '89999999999',
        avatar: '',
      };

      const { asFragment } = render(<PlayerInfo data={testUser} />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
