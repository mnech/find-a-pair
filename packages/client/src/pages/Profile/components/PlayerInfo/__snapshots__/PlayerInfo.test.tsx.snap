// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerInfo should be rendered correctly PlayerInfo component default 1`] = `
<DocumentFragment>
  <div
    class="player-info"
  >
    <div
      class="group"
    >
      <span>
        Имя
      </span>
      <span>
        Иван
      </span>
    </div>
    <div
      class="group"
    >
      <span>
        Фамилия
      </span>
      <span>
        Иванов
      </span>
    </div>
    <div
      class="group"
    >
      <span>
        Имя профиля
      </span>
      <span>
        Иванушка
      </span>
    </div>
    <div
      class="group"
    >
      <span>
        Логин
      </span>
      <span>
        ivanovv
      </span>
    </div>
    <div
      class="group"
    >
      <span>
        Почта
      </span>
      <span>
        ivanov@yandex.ru
      </span>
    </div>
    <div
      class="group"
    >
      <span>
        Телефон
      </span>
      <span>
        89999999999
      </span>
    </div>
  </div>
</DocumentFragment>
`;
